trackers/__init__.py

```diff
@@ -1,5 +1,14 @@
-from trackers.core.deepsort.feature_extractor import DeepSORTFeatureExtractor
-from trackers.core.deepsort.tracker import DeepSORTTracker
 from trackers.core.sort.tracker import SORTTracker
 
-__all__ = ["DeepSORTFeatureExtractor", "DeepSORTTracker", "SORTTracker"]
+__all__ = ["SORTTracker"]
+
+try:
+    from trackers.core.deepsort.feature_extractor import DeepSORTFeatureExtractor
+    from trackers.core.deepsort.tracker import DeepSORTTracker
+
+    __all__.extend(["DeepSORTFeatureExtractor", "DeepSORTTracker"])
+except ImportError:
+    print(
+        "DeepSORT dependencies not installed. DeepSORT features will not be available."
+    )
+    pass
```

Comment:
Nice!

Reply:
@soumik12345 this is great, but I'd also want to include a recommendation on what to do to fix it. Here's [how](https://github.com/roboflow/rf-detr/blob/a9851a3f5fe611437663393453438ff1cc28c6a1/rfdetr/main.py#L482) we did it in RF-DETR.

Reply:
Yeah, IMO including a pip installation instruction would be helpful. We also need to author a strong installation instructions document.

Reply:
Probably we should also have installation instruction for every tracker in their respective docs.

Reply:
I included pip install method to except section 

```python
from trackers.core.sort.tracker import SORTTracker

__all__ = ["SORTTracker"]

try:
    from trackers.core.deepsort.feature_extractor import DeepSORTFeatureExtractor
    from trackers.core.deepsort.tracker import DeepSORTTracker

    __all__.extend(["DeepSORTFeatureExtractor", "DeepSORTTracker"])
except ImportError:
    print(
        "DeepSORT dependencies not installed. DeepSORT features will not be available. "
        "Please run `pip install trackers[deepsort]` and try again."
    )
    pass

```